src/fosslight_android/android_binary_analysis.py

```
@@ -193,21 +195,85 @@ def read_module_info_from_build_output_file():
+            try:
+                line = line.strip()
+                pattern = re.compile(r'.*PLATFORM_VERSION\s*=\s*(\d+)\.?\d*\S*\s*')
+                matched = pattern.match(line)
+                if matched is not None:
+                    platform_version = matched.group(1)
+                    break
 
-        except Exception:
-            pass
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)